src/DependencyInjection/Configuration.php

```diff
@@ -243,6 +243,7 @@ private function addFilesystemSection(ArrayNodeDefinition $node)
                                     ->end()
                                 ->end()
                                 ->scalarNode('region')->defaultValue('s3.amazonaws.com')->end()
+                                ->scalarNode('endpoint')->defaultNull()->end()
```

Comment:
please add `->info()` node with a meaningful description

Reply:
if this good description or not

`->scalarNode('endpoint')->defaultNull()->info('custom bucket endpoint for any storage use s3 SDK like DigitalOcean Spaces')->end()`